terraform/modules/helm/helm.tf

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+data "oci_containerengine_clusters" "oke_clusters_list" {
+  compartment_id = var.oke_compartment_ocid
+}
+
+locals {
+  oke_clusters_list = data.oci_containerengine_clusters.oke_clusters_list.clusters
+  oke_cluster_name  = [for c in local.oke_clusters_list : c.name if c.id == var.oke_cluster_ocid][0]
+
+  helm_inputs = {
+    # global
+    "global.namespace"           = var.deploy_mushop_config ? "livelab-test" : var.kubernetes_namespace
+    "global.kubernetesClusterID" = var.oke_cluster_ocid
+
+    # oci-onm-logan
+    "oci-onm-logan.ociLANamespace"        = var.oci_la_namespace
+    "oci-onm-logan.ociLALogGroupID"       = var.oci_la_logGroup_id
+    "oci-onm-logan.image.url"             = var.container_image_url
+    "oci-onm-logan.kubernetesClusterName" = local.oke_cluster_name

```

Comment:
Set this at global.

Reply:
**kubernetesClusterName** was set to global scope. 
Validation - Stack is successfully deploying post this change.